### PR TITLE
Reorganizing and rewording docs.

### PR DESCRIPTION
**Pull request by dblotsky**
- Put examples below events
- De-duped returned object section
- Removed Tizen (deprecated)
- Made quirks a little easier to distinguish
- Switched to using reference-style links

### CODE REVIEW

**Comment by dblotsky** [comment]
@nikhilkh @riknoll @rkatyal please review when you have a moment.

File: README.md [hunk 1]
```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
...
-### Windows Phone 7 and 8 Quirks
+### Quirks: Windows Phone 7 &amp; Windows Phone 8
 
-Windows Phone 7 does not provide native APIs to determine battery
-level, so the `level` property is unavailable.  The `isPlugged`
-parameter _is_ supported.
+The `level` property is _not_ supported on Windows Phone 7 because the OS does not provide native APIs to determine battery level. The `isPlugged` parameter _is_ supported.
 
-### Windows Quirks
+### Quirks: Windows &amp; Windows Phone 8.1
```
Comment (nikhilkh) [inline README.md]: I think the quirk only applies to Windows Phone 8.1 as stated and not to Windows platform overall which includes Windows 8.1, Windows Phone 8.1 & Windows 10. Does your edit change the meaning?

Comment (nikhilkh) [inline README.md]: Scratch that - I see what you have done - I'm OK with it. Let's get this in. It LGTM.

File: README.md [hunk 2]
```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
...
 
 - Amazon Fire OS
 - iOS
 - Android
 - BlackBerry 10
-- Tizen
 - Firefox OS
 - Windows (Windows Phone 8.1 only)
-- Browser
+- Browsers
```
Comment (riknoll) [inline README.md]: Browser is the name of the platform. For the supported ones, maybe just list them in parentheses like windows does above

File: README.md [hunk 3]
```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
...
 - iOS
 - Android
 - BlackBerry 10
-- Tizen
 - Firefox OS
 - Windows (Windows Phone 8.1 only)
-- Browser
-
-### Windows Quirks
+- Browsers
```
Comment (riknoll) [inline README.md]: Browser is the name of the platform. For the supported ones, maybe just list them in parentheses like windows does above

File: README.md [hunk 4]
```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
 - BlackBerry 10
 - Windows Phone 7 and 8
 - Windows (Windows Phone 8.1 only)
-- Tizen
 - Firefox OS
-- Browser
+- Browsers
```
Comment (riknoll) [inline README.md]: Browser is the name of the platform. For the supported ones, maybe just list them in parentheses like windows does above

File: README.md (hunk 1 above)
Comment (dblotsky) [inline README.md]: That's still a good point. I fixed it to be more specific.

**Pull request merged**